### ISSUE TITLE
Lot of Unknown target inventory errors on Paper-318 (MC: 1.20.2)

### ISSUE THREAD

**Issue by lerokko** [Mille12313/MuleTracker#1]
Downloaded the latest file from spigot (1.3) but `/paper version` still says 1.2
```
- [Sat 10:47:04 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftInventoryEnchanting
- [Sat 10:47:04 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftBlockInventoryHolder
- [Sat 10:47:10 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftInventoryEnchanting
- [Sat 10:47:10 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftBlockInventoryHolder
- [Sat 10:47:14 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftInventoryGrindstone
- [Sat 10:47:14 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftBlockInventoryHolder
- [Sat 10:47:22 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftInventoryEnchanting
- [Sat 10:47:22 ERROR MuleTracker] Unknown target inventory type: org.bukkit.craftbukkit.v1_20_R2.inventory.CraftBlockInventoryHolder
```

This is just a small excerpt.
Edit: You said on spigot to raise issues on your discord so I can also do that if you prefer that channel.

**Comment by namp007** [comment]
We'll look into it, thanks for reporting

**Comment by namp007** [comment]
Fixed in 1.3.1
